test/test_serde.py

```diff
@@ -187,14 +189,22 @@ def test_pointer_tensor_simplify(self):
 class TestSerde(object):
     @pytest.mark.parametrize("compress", [True, False])
     def test_torch_Tensor(self, compress):
-
         hook = TorchHook(torch)
 
         t = Tensor(numpy.random.random((100, 100)))
         t_serialized = serialize(t, compress=compress)
         t_serialized_deserialized = deserialize(t_serialized, compressed=compress)
         assert (t == t_serialized_deserialized).all()
 
+    @pytest.mark.parametrize("compress", [True, False])
+    def test_torch_Tensor_convenience(self, compress):
```

Comment:
Can you add a comment detailing what this is testing? What's the difference between this test and `test_hooked_tensor`?

Reply:
good call

Reply:
done